nvflare/lighter/impl/cert.py

```diff
@@ -125,7 +125,7 @@ def build(self, project, ctx):
 
     def get_pri_key_cert(self, participant):
         pri_key, pub_key = self._generate_keys()
-        subject = participant.subject
+        subject = self.get_subject(participant)
```

Comment:
instead of wrapping "participant.subject" into a method,

suggest just adding type hint `participant: Participant`